### ISSUE TITLE
earth leviathan / forest keepers

### ISSUE THREAD

**Issue by sangca2002** [ImpulsiveLad/BetterTotalScrap#1]
when those two monsters kill a person with scrap, the scrap gets removed from the game, resulting in them not being calculated.

**Comment by ImpulsiveLass** [comment]
Yeah there really is no way to count something that doesn't exist anymore, only objects that currently exist in the scene can be referenced in code. Once it is destroyed it won't be tracked as existing, the count is taken when the level is pulled. same issue I had with the apparatus in 1.0.0 as it was not a thing anymore when the scrap used to be counted

**Comment by sangca2002** [comment]
can't you fix it by registering the scrap when a player gets killed? It can be taken for account later when the calculation takes place... i have ocd too and i cant take vow having 150 worth scrap in total
